website/src/content/docs/ja/index.mdx

```diff
@@ -43,3 +43,22 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
         フロントエンド開発で利用される言語に対して、優れたサポートを提供します。特にTypeScriptおよびJSXへのサポートを第一にしています。
     </Card>
 </CardGrid>
+
+
+## Sponsors
+
+### Gold Sponsors
+
+<div class="sponsors">
+    <a class="sponsors-gold" href="https://shiguredo.jp/" rel="nofollow">
+      <img title="Shiguredō" src="https://camo.githubusercontent.com/58b1dd652b0e2bfb67b97b59c2785bd5694dc0e9e8f6ce2a4182eacc35ed1fec/68747470733a2f2f73686967757265646f2e6a702f6f6666696369616c5f73686967757265646f5f6c6f676f2e737667" data-canonical-src="https://shiguredo.jp/official_shiguredo_logo.svg"  />
+    </a>
+</div>
+
+### Bronze Sponsors
+
+<div class="sponsors">
+    <a class="sponsors-bronze" href="https://kanamekey.com" rel="nofollow">
+      <img title="KANAME" src="https://camo.githubusercontent.com/3a726b47a0797ddecdb6474cec6418fdf98ffc59d4f17327bdab13dbc079d90c/68747470733a2f2f696d616765732e6f70656e636f6c6c6563746976652e636f6d2f6b616e616d652f643135666439382f6c6f676f2f3235362e706e673f6865696768743d3830" data-canonical-src="https://images.opencollective.com/kaname/d15fd98/logo/256.png?height=256"  />
+    </a>
+</div>
```

Comment:
Bronze sponsors can't be shown to our website because it isn't part of our perks: https://opencollective.com/biome

Bronze sponsor are only featured in our README

Reply:
Oh, Alright